amp/system-oracle/README.md

```diff
@@ -48,7 +49,20 @@ $ oc patch dc/system-app -p '[{"op": "replace", "path": "/spec/strategy/rollingP
 $ oc patch dc/system-app -p '[{"op": "add", "path": "/spec/strategy/rollingParams/pre/execNewPod/env/-", "value": {"name": "ORACLE_SYSTEM_PASSWORD", "value": "<<<<SYSTEM_PASSWORD>>>>"}}]' --type=json
```

Comment:
This will probably need to change, because for some reason the post hook fails with following error message
`Failed to connect to database: Environment ORACLE_SYSTEM_PASSWORD is mandatory (KeyError)`
It can be fixed (I know that it might not be desired) by setting the env variable
`oc set env dc/system-app ORACLE_SYSTEM_PASSWORD=<<<<SYSTEM_PASSWORD>>>>`

Reply:
I've found maybe better solution:
```
oc patch dc/system-app -p '{"spec": {"strategy": {"rollingParams": {"post":{"execNewPod": {"env": [{"name": "ORACLE_SYSTEM_PASSWORD", "value": "<<<<SYSTEM_PASSWORD>>>>"}]}}}}}}'

Reply:
OK then it is needed in `post` too